internal/replace.go

```diff
@@ -180,3 +180,42 @@ func hasExtraDiv(n *html.Node) bool {
 
 	return false
 }
+
+// TextFormatter catches Bold, Italic or underlined formatting
+type TextFormatter struct{}
+
+// ReplaceTag implements the TagReplacer interface
+func (*TextFormatter) ReplaceTag(n *html.Node) {
+	if isBold(n) {
+		n.Data = "strong"
+		n.Attr = []html.Attribute{}
+	}
+	if isItalic(n) {
```

Comment:
With this logic, "italic" wins over "bold" for the input like `<span style="font-weight: bold; font-style: italic">`.  I'm okay to keep one overwriting the other instead of printing `__**text**__`, but it seems like "bold" should take precedence